commons/src/main/java/com/simpletools/calculator/commons/operations/LengthOperation.kt

```diff
@@ -0,0 +1,36 @@
+package com.simpletools.calculator.commons.operations
+
+object LengthOperation {
+    var value: Double = 0.0
+    var conversion: String = ""
+    var LengthMap = HashMap<String, Double>()
+
+    init {
+        LengthMap.put("cm cm", 1.0)
```

Comment:
why not use underscores as a separator ? Also I wonder if this can be put inside of a json file to be more easily maintained. The JSON could contain all three of the conversionMaps. Eliminating the need for three classes.

Reply:
That would reduce the cohesion of the class(es) and would violate our current operation 'design pattern' where operations are classes that perform a single thing.